MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -1466,6 +1467,7 @@
 		8DF8E4DD2202694100B29FEF /* Documents */ = {
 			isa = PBXGroup;
 			children = (
+				8A047B1125E87EA3001CE518 /* Package.swift */,
```

Comment:
I’m not sure if this is a good idea. There’s already a reference to this Package.swift one level up, between the Example and CMapboxCoreNavigation groups.

Does this extra reference ensure that the main, developer-facing Package.resolved and the internal-to-this-workspace Package.resolved stay in sync? Do you get any project integrity errors about projects or folders that can’t be loaded twice?

Reply:
Was added mostly for convenience, will remove to prevent any clashes with `Package.swift` located in `MapboxNavigation` package.